From: Helmut Grohne <helmut@subdivi.de>
Subject: Invoke the triplet prefixed pkg-config
Bug-Debian: http://bugs.debian.org/833171
Forwarded: https://sourceforge.net/p/autogen/bugs/179/
Origin: vendor
Last-Update: 2016-09-03

Otherwise pkg-config examines the build architecture which usually doesn't have
the relevant dependencies installed or whose flags is different.

Index: autogen-5.18.10/config/ag_macros.m4
===================================================================
--- config/ag_macros.m4
+++ config/ag_macros.m4
@@ -30,7 +30,7 @@
 AC_DEFUN([INVOKE_AG_MACROS_LAST],[
 [if test X${INVOKE_AG_MACROS_LAST_done} != Xyes ; then]
   GUILE_FLAGS
-  [ag_gv=`gdir=\`pkg-config --cflags-only-I \
+  [ag_gv=`gdir=\`$PKG_CONFIG --cflags-only-I \
   guile-${GUILE_EFFECTIVE_VERSION} | \
   sed 's/-I *//;s/ *-I.*/ /g'\`
   for d in $gdir

